tee-worker/litentry/core/vc-task/receiver/src/test.rs

```diff
@@ -0,0 +1,105 @@
+use core::time::Duration;
+
+use lc_vc_task_sender::{VCRequest, VcRequestSender};
+use litentry_primitives::AesRequest;
+
+use super::*;
+use crate::mock::*;
+
+#[test]
+#[ignore]
```

Comment:
Why is this test case ignored ?

Reply:
Yeah this is so that it is not run with all the tests, and only run with the `test-threads 1`

Reply:
This means if we add more `ignored` tests to this package they will be run alongside  ones added in this PR, is this fine ?

Reply:
Yeah that should be fine imo, they will be run with `cargo test -p lc-vc-task-receiver -- --ignored --test-threads 1 `